### PR TITLE
DDF-1692 Add per-source hit count to result properties

### PR DESCRIPTION
**Pull request by brendan-hofmann**
[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/373)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/373" target="_blank">ReviewNinja

**Comment by brendan-hofmann** [comment]
@pklinef @spearskw @AzGoalie

File: catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java [hunk 1]
```diff
@@ -138,8 +140,11 @@ public void run() {
                     processingDetails
                             .add(new ProcessingDetailsImpl(sourceId, new NullPointerException()));
                 } else {
+                    // use commons collections for this incrementing bullshit
```
Comment (rzwiefel) [inline catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java]: Not sure if you meant to leave this one in there.. :P

Comment (brendan-hofmann) [inline catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java]: Well...woops. ;)

**Comment by AzGoalie** [comment]
:+1:

**Comment by spearskw** [comment]
:+1:

**Pull request merged**